web/controllers/password_reset_controller.ex

```diff
@@ -12,7 +12,7 @@ defmodule CodeCorps.PasswordResetController do
   """
   def reset_password(conn, %{"token" => token, "password" => password, "password_confirmation" => password_confirmation}) do
     user = conn.assigns.current_user
-    with %AuthToken{value: auth_token} <- Repo.get_by(CodeCorps.AuthToken, %{ value: token, user_id: user.id }),
+    with %AuthToken{value: auth_token} <- Repo.get_by(AuthToken, %{ value: token, user_id: user.id }),
```

Comment:
I think this should probably `get_by` the `token` only, since we don't necessarily have the `user` here. It won't be in the `assigns` in this case.